Tests/Client/Unit Tests/WebSocket/WebSocketTests.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  WebSocketTests.swift
+//  StreamChatClientTests
+//
+//  Created by Alexey Bukhtin on 02/05/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+final class WebSocketTests: XCTestCase {
+    
+    let mockProvider = WebSocketProviderMock(request: URLRequest(url: URL(string: "http://test.com")!), callbackQueue: .main)
```

Comment:
It's a good practice to create these in every setUp() run so the tests always run in a clean environment.